packages/core/scripts/remove-collateral-upp/2_VoteSimulate.js

```diff
@@ -0,0 +1,260 @@
+// This script simulates a vote from a large token holder to ratify the proposal from script 1_Propose.js.
```

Comment:
This should probably just be a symlink like the other `2_VoteSimulate.js` like [this one](https://github.com/UMAprotocol/protocol/blob/master/packages/core/scripts/creator-umip/2_VoteSimulate.js)

Reply:
+1. no point in re-duplicating this file

Reply:
Maybe a dumb question, but how do I create this as a symlink?

Reply:
I just copied and pasted another `2_VoteSimulate.js` in my code editor (VSCode). Try that?

Reply:
Hmm, in Atom it actually copies the code into a new file instead of copying the symlink.

Reply:
Installed VSCode to copy the symlink and it worked.